cfme/tests/automate/test_ansible_tower_automate.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.config_management.ansible_tower import AnsibleTowerProvider
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log_validator import LogValidator
+
+pytestmark = [
+    test_requirements.automate,
+    pytest.mark.provider([AnsibleTowerProvider], scope='module'),
+    pytest.mark.usefixtures('setup_provider')
```

Comment:
This is a function scoped fixture, your provider marker is module scoped. Is this intentional? It will remove/add the provider between test cases in the module.  Right now there's one provider parametrized by our current configs, but when there are more this will cause a performance hit.